src/config/endpoints.js

```diff
@@ -0,0 +1,5 @@
+// URLs
+export const API_URL = 'https://datalyzer-api.herokuapp.com';
+
+// Endpoints
+export const LOGIN_ENDPOINT = `${API_URL}/auth/login`;
```

Comment:
Are you going to declare all endpoints here?
If so, will you repeat `${API_URL}` every time?

Reply:
1. Yes
2. What do you suggest to do?

Reply:
fixed